recipes/fml/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "fml" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/fwilliams/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 24891fbe9151cf1222d41d5cd53867b1f2cc01e6fc3f12f968b0ab19705c172f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytorch
```

Comment:
Is `pytorch` needed here? Often `python` and `pip` is enough in the `host` section.